transport/pool.go

```diff
@@ -129,12 +128,14 @@ type PoolRefiller struct {
 	active int
 }
 
+// Here addr will consist only of IP, because init is called on addresses
+// gathered from system.peers or system.local tables.
 func (r *PoolRefiller) init(addr string) error {
```

Comment:
I suggest you now replace `addr` with `host` in this file and its test.

Reply:
Can we have the following name convention?
- **host** is always IP only
- **addr** can be IP only or IP with port
We can still open connection to both host and addr, because in case of lacking port, it is automatically set to the default one.

Reply:
On the other hand, this convention won't hold in cluster, where we are asking Scylla for **rpc_address** and getting IP only.
Moreover, **Refiler** has field named **addr**, so should it be renamed to **host** even if it can contain port?